libbeat/template/template.go

```diff
@@ -128,7 +128,7 @@ func (t *Template) generate(properties common.MapStr, dynamicTemplates []common.
 	}
 
 	if t.esVersion.IsMajor(2) || t.esVersion.IsMajor(5) {
-		basicStructure.Put("mappings._default_._all.norms.enabled", false)
```

Comment:
@urso We also use `doc` for all types if we write to indices from 5.x with the ES output? Just want to make sure this also works with older versions of ES.